packages/web-components/src/components/cbp-list/cbp-list.tsx

```diff
@@ -44,9 +44,10 @@ export class CbpList {
       return (
         <Host>
           <Tag 
-            ref={(el) => this.renderedTag = el}
-            // aria-size={this.size} 
+            ref={(el) => this.renderedTag = el} 
             aria-label={this.accessibilityText}
+            aria-size={this.size} 
+            aria-variant={this.variant}
```

Comment:
this isn't a thing and should be deleted. This is what props are for. ARIA is not for making up fake attributes.